README.md

```diff
@@ -23,16 +28,31 @@ Examples:
 Options:
   -r, --raw   Output the link to the raw text
   -h, --help  Show help                      
-
 ```
 
-## Example
+#### Example
 
 ```sh
 $ echo "hi" | hastebin
 http://hastebin.com/ilitixevat
 ```
 
+---
+
+### Via the JavaScript API
+
+```js
+var hastebin = require('hastebin')
+
+hastebin.createPaste('url', {
```

Comment:
Can you change the example text from `'url'` to something more... text like? Took me a second to figure out that was the text content of the paste.

Reply:
Oh, sorry about that. I was inbetween two different things and forgot that that's the parameter for the *content*.

Reply:
Boom :D

Reply:
Pushed :+1: